app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/OverlayScreenshotManager.kt

```
@@ -0,0 +1,276 @@
+      }
+
+      val cacheDir = context.cacheDir
+      val screenshotFile = File(cacheDir, "screenshot_${System.currentTimeMillis()}.jpg")
+
+      FileOutputStream(screenshotFile).use { out ->
+        scaledBitmap.compress(Bitmap.CompressFormat.JPEG, SCREENSHOT_QUALITY, out)
+      }
+
+      scaledBitmap.recycle()
+      screenshotFile
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)